bedrock/products/templates/products/vpn/landing-refresh.html

```diff
@@ -32,11 +32,37 @@
 {% set _utm_campaign = 'vpn-product-page' %}
 {% set _params = '?utm_source=' ~ _utm_source ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign %}
 
+
+{% set show_events_banner = vpn_available_in_country(country_code) and ftl_has_messages('banner-vpn-holidays-take-20-percent-off', 'banner-vpn-holidays-use-coupon-code', 'banner-vpn-holidays-get-annual-sub', 'banner-vpn-holidays-offer-ends') and
+switch('vpn-events-banner') %}
+
+{% set holidays_na_banner_experiment = switch('vpn-holidays-na-test') and country_code in ['US', 'CA'] %}
+
+{% block experiments %}
+  {% if holidays_na_banner_experiment %}
+    {{ js_bundle('mozilla-vpn-holidays-na-experiment-banner') }}
+  {% endif %}
+{% endblock %}
+
 {% block page_css %}
   {{ css_bundle('protocol-split') }}
   {{ css_bundle('protocol-picto')}}
   {{ css_bundle('mozilla-vpn-common-refresh') }}
   {{ css_bundle('mozilla-vpn-landing-refresh') }}
+
+  {% if show_events_banner %}
+    {{ css_bundle('mozilla-vpn-events-banner') }}
+  {% endif %}
+
+  {% if show_holidays_na_banner %}
+    {{ css_bundle('mozilla-vpn-events-banner') }}
+  {% endif %}
```

Comment:
Issue: this CSS bundle is being loaded twice, as it's already on line 54 above?

Reply:
oops